conf/layer.conf

```diff
@@ -5,29 +5,93 @@ BBPATH := "${LAYERDIR}:${BBPATH}"
 BBFILES += "${LAYERDIR}/recipes-wolfssl/wolfssl/*.bb \
             ${LAYERDIR}/recipes-wolfssl/wolfssl/*.bbappend"
 
+BBFILES += "${@bb.utils.contains('IMAGE_INSTALL', \
+                                 'wolfssl', \
+                                 '${LAYERDIR}/recipes-wolfssl/wolfssl/*.bb ${LAYERDIR}/recipes-wolfssl/wolfssl/*.bbappend', \
+                                 '', d)}"
+
 #Comment out if you don't want to use wolfclu 
-BBFILES += "${LAYERDIR}/recipes-wolfssl/wolfclu/*.bb \
-            ${LAYERDIR}/recipes-wolfssl/wolfclu/*.bbappend"                                                 
+#BBFILES += "${LAYERDIR}/recipes-wolfssl/wolfclu/*.bb \
+#            ${LAYERDIR}/recipes-wolfssl/wolfclu/*.bbappend"
+
+BBFILES += "${@bb.utils.contains('IMAGE_INSTALL', \
+                                 'wolfclu', \
+                                 '${LAYERDIR}/recipes-wolfssl/wolfclu/*.bb ${LAYERDIR}/recipes-wolfssl/wolfclu/*.bbappend', \
+                                 '', d)}"
 
 # Uncomment if wanting to use example tests                                                 
 #BBFILES += "${LAYERDIR}/recipes-examples/*/*/*.bb \
 #            ${LAYERDIR}/recipes-examples/*/*/*.bbappend"
+BBFILES += "${@bb.utils.contains('IMAGE_INSTALL', \
```

Comment:
The above is too permissive of a search. It should have `wolfcryptbenchmark` and `wolfcrypttest` in it